scripts/redirect_url_utils.py

```diff
@@ -25,14 +56,34 @@
         <meta http-equiv="REFRESH" content="0.0;url={redirect_to_url}" />
     </head>
     <body></body>
-</html>'''
+</html>
```

Comment:
This line changes every redirect file that we already have in the repository. Can we leave it as it was?
<img width="1067" alt="image" src="https://github.com/okta/okta-help/assets/4158731/0bda7a80-eb6c-46bf-8bd7-1f0d3794c1a7">

we will get rid of all of this later

Reply:
sure, reverting